### PR TITLE
Add postal code to customer in one line

### PR DESCRIPTION
**Pull request by tohhengxing**
**New features**

- Ability to add postal code to customer
- Updated GUI

### CODE REVIEW

File: src/main/java/seedu/address/model/person/PostalCode.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+/**
+ * Represents a Person's postal code number in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidPostalCode(String)}
+ */
+public class PostalCode {
+
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Postal Code should be exactly 6 digits";
+    public static final String VALIDATION_REGEX = "\\d{6}";
```
Comment (rimgik) [inline src/main/java/seedu/address/model/person/PostalCode.java:15]: I think postal need not to be 6 digits.

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 2]
```diff
@@ -11,5 +11,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_POSTAL_CODE = new Prefix("pc/");
```
Comment (rimgik) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:14]: Can we shorten prefix?

**Review by rimgik — CHANGES_REQUESTED** [review]
Overall, the changes look solid! I just have a few minor suggestions for improvement.

File: src/main/java/seedu/address/model/person/PostalCode.java (hunk 1 above)
Comment (rimgik) [reply src/main/java/seedu/address/model/person/PostalCode.java:15]: But I believe our target user does not need to be in Singapore.

**Comment by yyanyee** [comment]
Looks good! Conflict resolved

**Pull request merged**